modules/lvms-deleting-lvmcluster-using-cli.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lvms-deleting-lvmcluster-using-cli_{context}"]
+= Deleting an LVMCluster CR by using the CLI
+
+You can delete the `LVMCluster` custom resource (CR) using the OpenShift CLI (`oc`). Upon deleting the LVMCluster CR, LVM Storage deletes the following custom resources:
+
+* `LVMVolumeGroup`
+* `LVMVolumeGroupNodeStatus`
+* `storageClass`
+* `volumeSnapshotClass`
+
+.Prerequisites
+
+* You have access to {product-title} as a user with `cluster-admin` permissions.
```

Comment:
The same prerequisites for uninstallation apply here as well. Removing the PVCs, snapshots, etc.

Reply:
Updated as per the suggestion.